ios/Podfile

```diff
@@ -0,0 +1,42 @@
+# Uncomment this line to define a global platform for your project
```

Comment:
It seems like this file wasn't there earlier. I added it & updated it (of-course using flutter)